src/main/java/edu/greenblitz/pegasus/commands/handleBalls/HandleBalls.java

```diff
@@ -0,0 +1,68 @@
+package edu.greenblitz.pegasus.commands.handleBalls;
+
+import edu.greenblitz.pegasus.commands.funnel.ReverseRunFunnel;
+import edu.greenblitz.pegasus.commands.funnel.RunFunnel;
+import edu.greenblitz.pegasus.commands.intake.IntakeCommand;
+import edu.greenblitz.pegasus.commands.intake.roller.ReverseRunRoller;
+import edu.greenblitz.pegasus.commands.shooter.ShooterEvacuate;
+import edu.greenblitz.pegasus.subsystems.Indexing;
+import edu.greenblitz.pegasus.utils.DigitalInputMap;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class HandleBalls extends IntakeCommand {
+
+
+	/**
+	 * this command checks if an enemy ball is trying to enter the Funnel,
+	 * and if so it is rejecting it from entering the funnel system using a
+	 * color sensor to check the color
+	 *
+	 * @see com.revrobotics.ColorSensorV3 colorSensor.
+	 * @see Indexing Subsystem
+	 */
+
+	private final Indexing index;
+	private boolean isBallInFunnel;
+
+	public HandleBalls() {
+		this.index = Indexing.getInstance();
+		isBallInFunnel = false;
+		require(Indexing.getInstance());
+	}
+
+	@Override
+	public void execute() {
+		SmartDashboard.putBoolean("isBallInFunnel", isBallInFunnel);
+		SmartDashboard.putBoolean("isEnemyBallUnSensor", index.isEnemyBallInSensor());
```

Comment:
```suggestion
		SmartDashboard.putBoolean("isEnemyBallInSensor", index.isEnemyBallInSensor());
```

change the original name too, cause uh you have  atypo